.jenkins/infrastructure/e2e_testing.Jenkinsfile

```diff
@@ -99,9 +102,12 @@ pipeline {
                     string(name: 'UBUNTU_2004_CFL_CUSTOM_LABEL', value: params.UBUNTU_2004_CFL_LABEL),
                     string(name: 'UBUNTU_2004_ICX_CUSTOM_LABEL', value: params.UBUNTU_2004_ICX_LABEL),
                     string(name: 'UBUNTU_2004_NONSGX_LABEL', value: params.UBUNTU_NONSGX_LABEL),
-                    string(name: 'WS2019_DCAP_CFL_LABEL', value: params.WINDOWS_2019_DCAP_LABEL),
```

Comment:
Please revert the name (not the values) changes back to what you had before as those were correct.

The names specifies the names of the parameters used in the openenclave-testing pipeline. The values are the parameters specified in this pipeline (i.e. lines 27-46 in this file).

What this section does is pass specific parameters from this pipeline into OpenEnclave-Testing pipeline to run, which is why the names are similar but different. The inconsistency can be fixed by changing all occurrences to be prefixed with either `WS*` or `WINDOWS_*` or something else, but that is outside the scope of this PR.